src/windows/map.c

```diff
@@ -162,6 +162,14 @@ static rct_window_event_list window_map_events = {
 	window_map_scrollpaint
 };
 
+/** rct2: 0x00F1AD61 */
+static uint8 _activeTool;
+
+/** rct2: 0x00F1AD6C */
+static uint32 _curPixel;
```

Comment:
I think its current line. It draws it over multiple game ticks so it stores it statically.